custom_components/tuya_local/devices/woox_r4028_powerstrip.yaml

```diff
@@ -0,0 +1,83 @@
+name: Woox R4028 + DIGOO DG-PS01
+primary_entity:
+  entity: switch
+  name: switch 1
+  class: switch
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+secondary_entities:
+  - entity: switch
+    name: switch 2
+    class: switch
+    dps:
+      - id: 2
+        type: boolean
+        name: switch
+  - entity: switch
+    name: switch 3
+    class: switch
+    dps:
+      - id: 3
+        type: boolean
+        name: switch
+  - entity: switch
+    name: switch USB
```

Comment:
I think just USB is better here.  Natural English word order would be "USB switch", but the switch is redundant since the name is attached to a switch entity.